.github/workflows/release.yml

```diff
@@ -9,6 +9,39 @@ permissions:
   contents: read
 
 jobs:
+
+  publish-docker-images:
+    strategy:
+      matrix:
+        include:
+          - service: coupon
+          - service: frontend
+          - service: inventory
+          - service: membership
+          - service: pricing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Store new version in env
+        run: echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and Push Docker Image for ${{ matrix.service }}
+        uses: docker/build-push-action@v5
+        with:
+          push: true
+          tags: keyval/odigos-${{ matrix.service }}:${{ env.TAG }}
```

Comment:
Maybe match the package name: 'odigos-demo-<service>'?

Reply:
good catch